grpc-proto/src/main/java/io/stargate/grpc/Values.java

```diff
@@ -100,6 +100,10 @@ public static Value of(LocalTime value) {
     return Value.newBuilder().setTime(value.toNanoOfDay()).build();
   }
 
+  public static Value of(CqlDuration duration) {
+    return Value.newBuilder().setDuration(ByteString.copyFrom(duration.encode())).build();
```

Comment:
The duration is encoded directly on the client.
Alternatively, we could use a protobuf message with `months`, `days` and `nanos` fields, but since we'll need this encoding when we eventually send the value to Cassandra, I figured we might as well pass it all the way through.

Reply:
That's less usable from other languages/platforms, but if this is an internal API maybe it's not a big deal.

Reply:
I had the same concern: if it is not exposed to clients (i.e. gRPC public access is not affected) it's fine to have as implementation detail. But if decoding/encoding is needed everywhere, a simple scheme might be preferable.

But these types are shared between public gRPC and Bridge aren't they?